packages/create-block/lib/scaffold.js

```diff
@@ -119,11 +125,22 @@ module.exports = async (
 		info( '' );
 		code( '  $ npm run packages-update' );
 		info( '    Updates WordPress packages to the latest version.' );
+	}
+	info( '' );
+	info( 'To enter the folder type:' );
+	info( '' );
+	code( `  $ cd ${ slug }` );
+	if ( wpScripts ) {
+		info( '' );
+		info( 'You can start development with:' );
+		info( '' );
+		code( '  $ npm start' );
+	}
+	if ( wpEnv ) {
 		info( '' );
-		info( 'You can start by typing:' );
+		info( 'You can start WordPress with:' );
 		info( '' );
-		code( `  $ cd ${ slug }` );
-		code( `  $ npm start` );
+		code( '  $ npx wp-env start' );
```

Comment:
@noahtallen or @noisysocks, I chose to install `@wordpress/env` as a local package:
https://github.com/WordPress/gutenberg/blob/master/packages/env/README.md#installation-as-a-local-package

The documentation recommends putting it inside the `scripts` section. I skipped that in favor of:
`npx wp-env start`

I'm not sure what would be the best way here. I proposed we make `wp-env` integration with Create Block optional. How would you approach it? Global or local installation? Can you install it globally from the Node process in the first place?

Reply:
Global install works via `npm i -g @wordpress/env`, which is then accessible on the CLI as `wp-env`

That said, for local projects which you know will support wp-env for the environment, I do like installing it locally. Then, you know exactly what version is available for everyone, and no one has to do yet another extra set up command, since wp-env is included already.

Reply:
@noahtallen, thank you very much for the clarification. Everything is configured properly in that case 😄